cvat/apps/engine/views.py

```diff
@@ -256,6 +256,12 @@ def perform_create(self, serializer, **kwargs):
             organization=self.request.iam_context['organization']
         )
 
+    def perform_destroy(self, instance):
```

Comment:
I will not recommend implementing the logic this way. It is a workaround.

Need to optimize handers of signals. Manually call post_delete for tasks from post_delete for projects with all necessary prefetches. The same for post_delete jobs, etc. Need to use local memcache to check that the signal handler has been called already. The optimization should be trivial. It will work for all objects (not only for jobs)